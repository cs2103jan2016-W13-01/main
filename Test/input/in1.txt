add go to library
add go to gym
add go to lecture 04/05/2016 10am
add submit paper 04/12/2016 11:59pm
display
delete 3
delete 2