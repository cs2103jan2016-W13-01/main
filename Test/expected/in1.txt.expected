Task added successfully: go to library  Once
1. incomplete; go to library; -; -; Once
Task added successfully: go to gym  Once
1. incomplete; go to gym; -; -; Once
2. incomplete; go to library; -; -; Once
Task added successfully: go to lecture by 10:00 05 Apr Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to gym; -; -; Once
3. incomplete; go to library; -; -; Once
Task added successfully: submit paper by 23:59 12 Apr Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; submit paper; -; 23:59 12 Apr; Once
3. incomplete; go to gym; -; -; Once
4. incomplete; go to library; -; -; Once
Invalid command. Please try again.
Task list not loaded
Task deleted: go to gym  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; submit paper; -; 23:59 12 Apr; Once
3. incomplete; go to library; -; -; Once
Task deleted: submit paper by 23:59 12 Apr Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once

